### ISSUE TITLE
#8134 [a+] add anchors to the timeline on a+ : Position in Umfrage Modul not set correctly

### ISSUE THREAD

**Issue by sblindow** [liqd/adhocracy-plus#2605]
**URL:**
[Test Project](https://aplus-dev.liqd.net/permanent-test-project/projects/multimodul-timeline/)

**expected behaviour:**
When viewing the timeline on a device with a screen size small enough to necessitate repositioning, the default anchor position should automatically adjust for each event/ module clicked within the timeline.

**behaviour:** 
The default anchor position adjusts correctly for other modules/events on smaller screens, but not for the Survey Module. To replicate this issue, use the following steps:
1. Access the link provided above.
2. Set your browser to the maximum zoom level that still correctly displays the first chronological event.
3. Navigate to the "Umfrage Module (2. Online Beteiligung)."

Despite the page being sufficiently long, the Survey Module does not align to the correct position as expected.

**important screensize:**
3360x1890, 120% browser zoom

**device & browser:**
MacOs, Firefox 125.0.3 (64-Bit)

**Comment by TeaJayyy** [comment]
@goapunk Think this is not the case with the poll module: https://aplus-dev.liqd.net/permanent-test-project/projects/multimodul-timeline/?initialSlide=2#timeline-carousel
If you click the spot on timeline with the poll the anchor doesn't work although the page is long enough. I believe it has to do with the content of the poll module. Not an expert but might it be possible that the anchor activates first and then the content of the module is loaded?

**Comment by goapunk** [comment]
yes, with content loaded via react this won't work I guess, we'd need to find a different solution there
